scripts/gen.py

```diff
@@ -68,6 +68,7 @@
 _CPP_HEADER = """// Autogenerated file by {}. Do not edit directly!
 
 #include "aten_xla_bridge.h"
+#include <ATen/ExtensionBackendRegistration.h>
```

Comment:
Let's add this when the PR goes in, as otherwise this won't build.

Reply:
`Backend::XLA` doesn't exist yet either. How about not generating this when `generate_class` is set?

Reply:
Actually, this header wouldn't be generated in that mode, so we're good.

Reply:
SG